src/core/Str.pm

```diff
@@ -229,6 +243,7 @@ my class Str does Stringy { # declared in BOOTSTRAP
           nqp::p6box_i($i)
         )
     }
+    multi method index(Str:D: Cool:D $needle, Num:D $pos) {self.index: $needle.Str, $pos.Int}
```

Comment:
Not sure where this candidate came from. It did not exist before this PR. Did you mean `Cool:D` instead?